tests/testthat/test-table_lbt05.R

```diff
@@ -1,8 +1,7 @@
-library(scda)
-library(dplyr)
+adsl <- adsl_raw
 
 get_adlb <- function() {
-  adlb <- synthetic_cdisc_data("rcd_2022_02_28")$adlb # nolintr
+  adlb <- adlb_raw # nolintr
```

Comment:
```suggestion
  adlb <- adlb_raw # nolint
```

we don't need the "r"